packages/ptokens-entities/src/ptokens-asset-provider.ts

```diff
@@ -7,7 +7,7 @@ export abstract class pTokensAssetProvider {
    * @param _pollingTime - The polling period.
    * @returns A Promise that resolve with the same transaction hash __txHash_.
    */
-  abstract waitForTransactionConfirmation(_txHash: string, _pollingTime?: number): Promise<string>
+  abstract waitForTransactionConfirmation(_txHash: string): Promise<string>
```

Comment:
viem waitForTransactionReceipt supports an optional parameter pollingInterval. I wouldn't modify the signature

Reply:
I added the number of confiramtions to wait as parameter.